src/Cogworks.CogFlare.Core/Components/ExternalCachePurgeComponent.cs

```diff
@@ -1,126 +1,43 @@
-﻿using RelationTypes = Umbraco.Cms.Core.Constants.Conventions.RelationTypes;
-
-namespace Cogworks.CogFlare.Core.Components;
+﻿namespace Cogworks.CogFlare.Core.Components;
 
 public class ExternalCachePurgeComponent : 
     INotificationAsyncHandler<ContentPublishedNotification>,
     INotificationAsyncHandler<ContentDeletedNotification>,
     INotificationAsyncHandler<ContentUnpublishingNotification>,
     INotificationAsyncHandler<MediaSavedNotification>
 {
-    private readonly ICloudFlareCachePurgeService _cloudFlareCachePurgeService;
-    private readonly IUmbracoContentNodeService _umbracoContentNodeService;
-    private readonly IRelationService _relationService;
-    private readonly CogFlareSettings _cogFlareSettings;
-    private readonly ILogger<ExternalCachePurgeComponent> _logger;
+    private readonly ICachePurgeService _cachePurgeService;
```

Comment:
I highly encourage you to introduce a primary constructor 😊

Reply:
Yup if we use latest .NET - also recommend 🙌🏻